.travis.yml

```diff
@@ -10,7 +13,7 @@ go:
 go_import_path: k8s.io/kops
 
 script:
-  - GOPROXY=https://proxy.golang.org make nodeup examples test
+  - GOPROXY=https://proxy.golang.org travis_wait 20 make nodeup examples test
```

Comment:
why is this wait required? just curious

Reply:
For some reason there is no output for too much time in ARM tests and they time-out. It may be related to some changes Travis folks did, as it used to work much faster a few days ago.
https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received

Reply:
This is one of the failed tests: https://travis-ci.org/github/kubernetes/kops/builds/676536208